06_modules_lab/01.py

```diff
@@ -1,4 +1,19 @@
-""" Write a program that takes a count
-from sys.argv import and prints "Hello Python"
-count times
-"""
+#Commited by Shlomit
+import sys
+if(len(sys.argv)) < 2:
+    print "error - you didn't add any arguments!"
+    sys.exit
+
+loopCount = sys.argv[1]

```

Comment:
1. Use lowercase variable names
2. Better to convert to int here: `loop count = int(sys.argv[1])`